docs/kyma-environment-broker/03-01-service-description.md

```diff
@@ -238,3 +238,7 @@ The mapping between the platform region and the provider region (Azure, AWS or G
 ## Free plan
 
 Free plan allows you to install Kyma on Azure or AWS. The configurable provisioning parameters are the same as for the trial plan.
+
+___
+
+<a name="update"><sup>1</sup> this parameter is available for `PATCH` as well and can be updated with the same constraints as during provisioning</a>
```

Comment:
Alternatively, we can duplicate the relevant part of each table and make sections regarding updating, but I think providing the information as footnote is the least invasive to the text.